cmake/Corrosion.cmake

```diff
@@ -1462,7 +1462,7 @@ function(corrosion_add_cxxbridge cxx_target)
     # No suitable version of cxxbridge was installed, so use custom target to build correct version.
     if(NOT cxxbridge)
         if(NOT TARGET "cxxbridge_v${cxx_required_version}")
-            add_custom_command(OUTPUT "${CMAKE_BINARY_DIR}/corrosion/cxxbridge_v${cxx_required_version}/bin/cxxbridge"
+            add_custom_command(OUTPUT "${CMAKE_BINARY_DIR}/corrosion/cxxbridge_v${cxx_required_version}/bin/cxxbridge${CMAKE_EXECUTABLE_SUFFIX}"
```

Comment:
Thanks for reporting! However, I believe that `CMAKE_EXECUTABLE_SUFFIX`  gives the suffix for the target platform, but we need the executable suffix for the host platform (important when cross-compiling to a different OS).

Reply:
You're right, I found existed workaround in this project and just copied it :)